cmd/up/up.go

```diff
@@ -503,6 +508,24 @@ func loadManifestOverrides(dev *model.Dev, upOptions *UpOptions) error {
 	return nil
 }
 
+func setSyncDefaultsByDevMode(dev *model.Dev, fs afero.Fs) error {
```

Comment:
Can we add a test for this new function?

Reply:
done b4f7ad31312400db69a6e935dfc5b831526e9069